libsolidity/formal/SMTChecker.cpp

```diff
@@ -78,6 +78,9 @@ bool SMTChecker::visit(FunctionDefinition const& _function)
 		m_interface->reset();
 		m_pathConditions.clear();
 		m_expressions.clear();
+		m_specialVariables.clear();
```

Comment:
Hm, is this a bugfix? Or it just wasn't used before?

Reply:
The symbolic variables used by the special variables can live across transactions, that's why this wasn't there before, but I think it's safer to clear them anyway.